### PR TITLE
(WIP) Tutorial more details

### PR DESCRIPTION
**Pull request by justinlaughlin**
Adding some more equations/details to the tutorial

### CODE REVIEW

File: src/tutorial/fem.md [hunk 1]
```diff
@@ -65,16 +65,29 @@ non-conforming:
 
 <img src="../img/fem2.png" width="300">
 
-To solve for the unknown coefficients, we multiply Poisson's equation by another
-(test) basis function $\varphi_i$ and integrate by parts to obtain
+In order to solve for the unknown coefficients of $u_h$, we convert our equation
+into its <a href="../../fem_weak_form/">weak form</a> (also known as its *variational form*).
+To do so, we multiply the entire equation (1) by a _test function_, 
+$v \approx v_h = \sum_{i=1}^n c_i \varphi_i$, and integrate over the domain, $\Omega$:
```
Comment (cjvogl) [inline src/tutorial/fem.md:71]: To me, this is less clear than the original text b/c $c_j$ has already been defined as the degrees of freedom associated with $u_h$.  Maybe skip over the whole choice of test space and say 
> ... by the _test function_ $\varphi_i$ and integrate over the domain $\Omega$:

Comment (justinlaughlin) [reply src/tutorial/fem.md:71]: That is fair - I was thinking of using something like $d_i$, do you think that would make it clearer? The test basis functions were already defined using the same symbol with a different index which is why I chose $c_i$ but I agree it is confusing.

My thought process for including this section is that if I was seeing this for the first time I might think "why are $u$ and $v$ defined differently?"

Comment (tzanio) [reply src/tutorial/fem.md:71]: Maybe I am missing something, but for me this is less clear. 

I added the strong form (before integration by parts) and the references to the weak form and divergence theorem, but I don't understand why we need to test with a generic function $v$ and have double sums (the test space is linear, so testing with the basis functions $\varphi_i$ is sufficient).

Can you take a look at https://github.com/mfem/web/commit/e8b4df337f0fb0b315e786ad1dfba21ef8d44f76 and let me know if you'd like to see any additional changes

Comment (justinlaughlin) [reply src/tutorial/fem.md:71]: That looks a lot clearer. Thank you!

**Pull request closed without merging**